Update key-vault-ovw-storage-keys.md

fhokholdMSFT opened the pull request:
Added additional documentation to the managed storage account keys page. Bruce please review my changes. I put it in the current format to make it stand out as a sample on the page.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

fhokholdMSFT committed:
Update key-vault-ovw-storage-keys.md

Added additional documentation to the managed storage account keys page. Bruce please review my changes. I put it in the current format to make it stand out as a sample on the page.

BrucePerlerMS reviewed (approved):
There are a couple formatting nits I'd fix but will let it pass for now. Thanks for contributing!
